nx/lib/nx/lin_alg.ex

```diff
@@ -949,6 +949,46 @@ defmodule Nx.LinAlg do
     )
   end
 
+  @doc """
+  Calculates the Moore-Penrose inverse, or the pseudoinverse, of a matrix.
+
+  ## Options
+    * `:eps` - Rounding error threshold used to assume values as 0.
+
+  ## Examples
+
+      iex> pinv = Nx.LinAlg.pinv(Nx.tensor([[1, 1], [3, 4]]))
+      iex> pinv
```

Comment:
```suggestion
      iex> Nx.LinAlg.pinv(Nx.tensor([[1, 1], [3, 4]]))
```

Reply:
should be fixed in commit 14811dbec97df7007212336b3131042afa8079cc. deleted the extra line

Reply:
update: 4a8f75a0e0f9c22f7c87805b12ccd809332e1e9a